libsolidity/interface/StandardCompiler.cpp

```diff
@@ -489,6 +510,10 @@ Json::Value StandardCompiler::compileInternal(Json::Value const& _input)
 	if (errors.size() > 0)
 		output["errors"] = errors;
 
+	if (!m_compilerStack.unhandledSMTLib2Queries().empty())
+		for (string const& query: m_compilerStack.unhandledSMTLib2Queries())
+			output["auxiliaryInputRequested"]["smtlib2"]["0x" + keccak256(query).hex()] = query;
```

Comment:
I think this should be named `auxiliaryOutput`, potentially also `smtlib2queries` may be better than `smtlib2`.

Reply:
I like `smtlib2queries` better, `auxiliaryOutput` sounds vague.

Reply:
I actually meant `auxiliaryInputRequest.smtlib2queries` 😉